java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -938,6 +939,32 @@ public void strlen_returns_success() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
+    @Test
+    public void setrange_returns_success() {
+        // setup
+        String key = "testKey";
+        int offset = 42;
+        String str = "pewpew";
+        String[] arguments = new String[] {key, Integer.toString(offset), str};
+        Long value = 10L;
```

Comment:
Do we want this value to be accurate to what would actually be returned (`42 + str.length()`) or do we not really care since its just mocked anyways?

Reply:
I can put here even negative numbers, this test added for coverage and does not involve a real redis server

Reply:
Yeah was just checking, sounds good